src/test/java/com/wikia/webdriver/testcases/imageservingtests/ImageStorageTests.java

```diff
@@ -53,7 +54,7 @@ public void ImageStorage_001_deleteImage() {
         delete.getBannerNotifications().clickUndeleteLinkInBannerNotification();
     restore.giveReason(PageContent.CAPTION);
     restore.restorePage();
-    restore.getBannerNotifications().verifyNotificationMessage();
+    Assertion.assertTrue(restore.getBannerNotifications().isNotificationMessageVisible());;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)